platforms/hanwha-wn/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+include(ExternalProject)
+include(ProcessorCount)
+include(../shared/shared.cmake)
+
+set(BOOST_BUILD_PATH ${CMAKE_CURRENT_BINARY_DIR}/boost-prefix/src/boost)
+
+file(WRITE ${BOOST_BUILD_PATH}/user-config.jam "using gcc : arm : \"$ENV{WN}/arm-linux-gnueabi-g++\"")
+
+set (BOOST_ARCH_NAME boost_1_63_0.tar.bz2)
```

Comment:
Maybe we should try Boost libs we're using for SDK build. So we'll not have to deal with Boost downloading/build here.